Dronuts/client_src/src/components/customer/Menu.js

```diff
@@ -1,112 +1,141 @@
 import React, {Component} from 'react';
 import ReactDOM from 'react-dom';
+import axios from 'axios';
 
-var cart = []
+class Menu extends Component {
+  constructor(){
+    super();
+    this.state = {
+      menu: [],
```

Comment:
In general I agree with all the improvements you have made. I do worry however about implementing an empty menu before we implement how to add to the menu, as this would make it hard to test adding items to the cart and ordering if there are no items in the menu

Reply:
The database already has some menu items, so we can always use those data to test.